### PR TITLE
Applying license changes

### PR DESCRIPTION
**Pull request by sergenyalcin**
### Description of your changes

This PR applies license changes.

I have:

- [ ] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

Tested via Uptest: https://github.com/crossplane-contrib/provider-upjet-aws/actions/runs/8281184177

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <!--
-Thank you for helping to improve Official AWS Provider!
+Thank you for helping to improve Official Azuread Provider!
 
 Please read through https://git.io/fj2m9 if this is your first time opening a
-Official AWS Provider pull request. Find us in https://slack.crossplane.io/messages/dev if
-you need any help contributing.
+Official Azuread Provider pull request. Find us in https://crossplane.slack.com
```
Comment (mbbush) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: This is not Azuread

Comment (sergenyalcin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks @mbbush, done!

**Comment by sergenyalcin** [comment]
/test-examples="examples/ec2/v1beta1/vpc.yaml"

**Review by ulucinar — APPROVED** [review]
Thanks @sergenyalcin. lgtm. Looks like we need to add the license header back to upjet's setup.go.tmpl so that the license headers are present for the files internal/controller/*.go.

**Pull request merged**